toolbox/ui.py

```diff
@@ -137,7 +138,21 @@ def draw_umap_projections(self, utterances: Set[Utterance]):
         self.umap_ax.set_yticks([])
         self.umap_ax.figure.canvas.draw()
 
-    def setup_audio_devices(self,sample_rate):
+    def save_audio_file(self, wav, sample_rate):        
+        dialog = QFileDialog()
+        dialog.setDefaultSuffix(".wav")
+        fpath, _ = QFileDialog().getSaveFileName(
+            parent=self,
+            caption="Select a path to save the audio file",
+            filter="Audio Files (*.flac *.wav)"
```

Comment:
We should make the default (.wav) first in this list, on my system if I do not provide a file extension it appends .flac.